index.js

```diff
@@ -1,445 +0,0 @@
-        let stateForTable = roomLookUp[currentRoom]
-        console.log(stateForTable.desc);
-        zombieHorde()
-        console.log("You better hurry up they're getting closer!" + player.encroacment);
-        player.location = roomLookUp[currentRoom]
-      } else {
-        console.log('You can not make that move from' + currentRoom + 'to' + newRoom)
-      }
-    }
-
-    /*----------------------------------Story--------------------------------------------*/
-    //intro asking if the player would like to play
-    async function intro() {
-      const introMessage = `Welcome to the Zombie Apocalypse! Zombies are all around and closing in fast! Please word your actions in a [action] + [Item/Room] format`;
-      let startPrompt = await ask(introMessage + "\n" + "Do you understand?\n>_");
-      let cleanStart = cleanWords(startPrompt);
-      if (cleanStart === "yes") {
-        start();
-      } else {
-        console.log("Come back when you're ready then!");
-        process.exit();
-      }
-    }
-
-    intro();
-    // game start
-    async function start() {
-      const startMessage = `After some time running through woods, you come upon a gated community that looks overrun and long abandoned. The gate is losely held together by some rusted chains with a note that reads: “We have safety and supplies beyond the town through the tunnel, there is a key in one of the houses that will unlock the mansion at the end of the cul-de-sac and will give you access to the tunnel and make your way to Paradise Cove. Hurry, once the gates are open there is no way to close them and ‘they’ will get in!
-  
-  What would like you to do?`;
-
-      console.log(startMessage);
-      player.location = outside
-      while (player.encroachment > 0) {
-        let dirtyInput = await ask('>_')
-        let cleanInput = cleanWords(dirtyInput)
-        let cleanArray = cleanInput.split(' ')
-        let command = cleanArray[0]
-        let activity = cleanArray[cleanArray.length - 1]
-        if (cleanInput === 'i') {
-          if (player.location.inv.length === 0) {
-            console.log("There is nothing here")
-          } else {
-            player.location.inv.forEach(function (obj) {
-              console.log(obj)
-            })
-          }
-        } else if (cleanInput === 'c') {
-          if (player.inventory.length === 0) {
-            console.log("There is nothing in your pockets.")
-          } else {
-            player.inventory.forEach(function (obj) {
-              console.log(obj)
-            })
-          }
-        } else if (actions.move.includes(command)) {
-          changeRooms(activity)
-        } else if (actions.grab.includes(word)) {
-          takeItem(activity)
-        } else if (actions.consume.includes(word)) {
-          hordeGain(activity)
-        } else if (actions.unlock.includes(word)) {
-          unlock(activity)
-        } else if (actions.examine.includes(word)) {
-          examine(activity)
```

Comment:
These arguments being passed into your actions.move.includes(_argument_) are inconsistent and causing it to throw errors.